src/Accounts/Accounts/Account/ConnectAzureRmAccount.cs

```diff
@@ -598,6 +608,27 @@ public override void ExecuteCmdlet()
             }
         }
 
+        public static bool TryParseClaimsChallenge(string base64Input, out string claimsChallenge)
```

Comment:
Minor: consider making it private or move to a Utility class
```suggestion
        private static bool TryParseClaimsChallenge(string base64Input, out string claimsChallenge)
```